book/getting-started/admin-interface.rst

```diff
@@ -0,0 +1,109 @@
+The Sulu Admin Interface
+========================
+
+In the last steps you setup your own Sulu website. The admin area should already be accessible.
+It gives a nice impression of the features of Sulu. And that's just what this page does:
+Give you a short glance on the Sulu admin interface. Everything else you'll learn by exploring
+it. There won't be a dedicated documentation.
+
+
+Dashboard
+---------
+
+We got big plans for the dashboard, but at the moment it gives you a starting point with the 
+implemented search. Just enter whatever you want to work on and it will give you direct links.
+
+.. figure:: ../../img/admin-dashboard.png
+	:align: center
+
+
+Contacts
+--------
+
+Contacts on the one hand cover the handling of personal informations on the other hand they are also
+the user provider for Sulu Users.
+
+.. figure:: ../../img/admin-contact-people.png
+	:align: center
+
+You're also able to create organizations and connect people with them.
+
+.. figure:: ../../img/admin-contact-organization.png
+	:align: center
+
+
+Assets
+------
+
+Sulu saves assets like images or downloadable files in a global pool. They could be shared
+throughout webspaces.
+
+.. figure:: ../../img/admin-assets.png
+	:align: center
+
+
+Settings
+--------
+
+In the settings section, stuff that is closer to the system is defined. We got user roles,
+categories, tags and the cache.
+
+
+User Roles
+^^^^^^^^^^
+
+There are several pre configured roles. The admin interface looks like this:
```

Comment:
Actually there are no pre configured roles (depending on which `sulu:build` command you use there is maybe one admin role, but never several roles, unless you've created some on your own)